### PR TITLE
Polkadot Documentation

### PR DESCRIPTION
**Pull request by KitHat**
That's a PR that contains configuration for Polkadot Runtime documentation.

NB:
- We are currently using one of the templates as a documentation home. However, in future we will have more templates and documentation will migrate into its own repository.

Tasklist:
- [x] Generic Runtime Template released
- [x] Text for the main page is updated

### CODE REVIEW

File: components/home/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -29,6 +29,10 @@ xref:upgrades.adoc[[.card-title]#Upgrades# [.card-body]#pass:q[A comprehensive s
 --
 xref:contracts-cairo::index.adoc[[.card-title]#Contracts for Cairo# [.card-body]#pass:q[A library for secure smart contract development written in Cairo for Starknet.]#]
 --
+[.card.card-secondary.card-substrate-runtimes]
+--
+xref:substrate-runtimes::index.adoc[[.card-title]#Substrate Runtimes# [.card-body]#pass:q[A collection of runtime templates for parachain development on Polkadot]#]
```
Comment (ericglau) [inline components/home/modules/ROOT/pages/index.adoc]:
```suggestion
xref:substrate-runtimes::index.adoc[[.card-title]#Substrate Runtimes# [.card-body]#pass:q[A collection of runtime templates for parachain development on Polkadot.]#]
```

File: playbook.yml [hunk 2]
```diff
@@ -80,6 +80,10 @@ content:
     - url: https://github.com/OpenZeppelin/nile
       branches: release-v*
       start_path: docs
+    
+    - url: https://github.com/OpenZeppelin/polkadot-runtime-template
```
Comment (ericglau) [inline playbook.yml]: It looks like this redirects to https://github.com/OpenZeppelin/polkadot-generic-runtime-template.  Should this url be changed to https://github.com/OpenZeppelin/polkadot-generic-runtime-template ?

Comment (KitHat) [reply playbook.yml]: Good catch!
Will fix in a moment.

File: ui/preview/model.yml [hunk 3]
```diff
@@ -61,6 +61,16 @@ site:
...
       latestVersion: *latest_version_abc
+    substrate-runtimes:
+      name: substrate-runtimes
+      title: Substrate Runtimes
+      url: '#'
+      versions:
+      - &latest_version_runtime
+        url: '#' 
+        version: '0.1.x'
+        displayVersion: '0.1.x'
```
Comment (ericglau) [inline ui/preview/model.yml:72]: If the template is v1, should these be updated as well?

Comment (ericglau) [reply ui/preview/model.yml:72]: It looks like this file is for preview only, so this isn't necessary.

**Review by ericglau — APPROVED** [review]
LGTM.  Note: it looks like the version number in https://github.com/OpenZeppelin/polkadot-generic-runtime-template/blob/v1-after-audit/docs/antora.yml should be updated if it's intended to be v1.

**Comment by KitHat** [comment]
> LGTM. Note: it looks like the version number in https://github.com/OpenZeppelin/polkadot-generic-runtime-template/blob/v1-after-audit/docs/antora.yml should be updated if it's intended to be v1.

It was just updated :)

**Pull request merged**